### PR TITLE
Create vesktop

### PR DESCRIPTION
**Pull request by odomingao**

### CODE REVIEW

File: apparmor.d/profiles-s-z/vesktop [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  @{sys}/devices/@{pci}/usb@{int}/**/interface r,
+
+        @{PROC}/@{pid}/cmdline r,
+  owner @{PROC}/@{pid}/task/@{tid}/comm rw,
+
+  owner /dev/ r,
+
+  deny       /dev/tty rw,
+  deny owner /dev/tty@{int} rw,
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/vesktop:41]: Why do you deny them?

Comment (odomingao) [reply apparmor.d/profiles-s-z/vesktop:41]: As a silencer. It asks for these permissions but I don't think that it actually needs them, as everything on the program works fine, and AFAIK (correct me if I'm wrong) access to /dev/tty allows the program to escape confinement so I thought it was better to deny.

Comment (roddhjav) [reply apparmor.d/profiles-s-z/vesktop:41]: Do you have any source of this?

Comment (odomingao) [reply apparmor.d/profiles-s-z/vesktop:41]: For the possible /dev/tty escapes? https://www.errno.fr/TTYPushback.html and https://github.com/containers/bubblewrap/issues/142

**Comment by roddhjav** [comment]
Thanks, merged.

File: apparmor.d/profiles-s-z/vesktop (hunk 1 above)
Comment (roddhjav) [reply apparmor.d/profiles-s-z/vesktop:41]: Thanks. This has been discussed here: https://github.com/roddhjav/apparmor.d/discussions/340

Spoiler alert: the solution is not apparmor.

**Pull request merged**